CONTRIBUTING.md

```diff
@@ -15,6 +15,18 @@ You could:
 Head over to [react-server-test-pages](/packages/react-server-test-pages) and
 check out the README to get a test server set up.
 
+## Testing
+
+Yeah!  Do it!
```

Comment:
Hmm... `react-server-test-pages` doesn't actually have any automated tests ([yet](https://github.com/redfin/react-server/issues/147)).

But I guess that makes sense.